.gitignore

```diff
@@ -4,5 +4,3 @@
 /telegraf.exe
 /telegraf.gz
 /vendor
-.DS_Store
-process.yml
```

Comment:
Please don't touch the `.gitignore` file at all.

While I do agree `.DS_Store` should not be in there, please create a separate PR for that.

Reply:
First time doing git rebase. Well pull request is marked as draft so you should not be bothered.

Reply:
Okay, great! Let me know when you are ready.

Reply:
Ok managed to restore it. Thanks for noticing this.